### PR TITLE
Changelog for v2.1.0

### PR DESCRIPTION
**Pull request by azdagron**
Signed-off-by: Andrew Harding <aharding@vmware.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Changelog
+
+## [2.1.0] - 2022-04-29
+
+### Added
+- Added the workloadapi.WatchX509Bundles method which watches X.509 bundles from the Workload API (#192)
```
Comment (amartinezfayo) [inline CHANGELOG.md:6]: We usually don't include the "Added", "Changed", etc. word to the start of the phrase, as it's already included in the title of the section.

**Review by amartinezfayo — COMMENTED** [review]
Looks great!
Just a comment about how the phrases are constructed.

**Review by amartinezfayo — COMMENTED** [review]
I'm wondering if we should include #191? Persons developing on Windows may be interested to know that they can consume the library on Windows platform now, so it may be good to provide visibility on that.

**Comment by azdagron** [comment]
If I recall correctly, the library already worked on Windows. 191 just provided CI/CD pipeline support and fixed up some tests?

I'm happy to include it but I didn't originally because it didn't feel like something somebody would upgrade to this version for (since it already worked?).

**Comment by amartinezfayo** [comment]
That's fair. Code changes were only on unit tests.

**Review by amartinezfayo — APPROVED** [review]
🎉

**Pull request merged**